go.mod

```diff
@@ -3,6 +3,11 @@ module github.com/sonatype-nexus-community/cheque
 go 1.12
 
 require (
-	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
+	github.com/jedib0t/go-pretty/v6 v6.0.5
+	github.com/logrusorgru/aurora v2.0.3+incompatible
+	github.com/package-url/packageurl-go v0.1.0
+	github.com/shopspring/decimal v1.2.0
+	github.com/sirupsen/logrus v1.6.0
+	github.com/sonatype-nexus-community/go-sona-types v0.0.8
 	github.com/sonatype-nexus-community/nancy v0.0.21
```

Comment:
Do you want to eventually remove the `nancy` dep (by using go-sona-types)? If so, remove nancy dep? If not, maybe move nancy to `v1.0.1`?

Reply:
Definitely trying to remove the Nancy dependency. Good call!

Reply:
But it seems we still use some nancy stuff. Updated the dependency for now.